src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -330,10 +332,37 @@ private void addConfig(Map<String, ?> parsedProps, ComposableConfig config) {
     }
   }
 
+  /**
+   * Returns the url property. Preference is given to property <code>store.url</code> over
+   * <code>hdfs.url</code> because <code>hdfs.url</code> is deprecated.
+   *
+   * @param props - Map of properties for the connector
+   * @return String - url for HDFS
+   */
+  private String extractUrl(Map<String, String> props) {
+    String storageUrl = props.get(StorageCommonConfig.STORE_URL_CONFIG);
+    if (storageUrl != null && !storageUrl.equals(StorageCommonConfig.STORE_URL_DEFAULT)) {
+      return storageUrl;
+    }
+
+    String hdfsUrl = props.get(HDFS_URL_CONFIG);
+    if (hdfsUrl != null && !hdfsUrl.equals(HDFS_URL_DEFAULT)) {
+      return hdfsUrl;
+    }
+
+    throw new ConfigException(
+        String.format("Configuration %s cannot be empty."), StorageCommonConfig.STORE_URL_CONFIG
```

Comment:
your call to `format` has a bug. And the fact that we don't see it tells me we don't have good unit tests. 
`HdfsSinkConnectorConfigTest` is the place to add some.